module/VuFindSearch/src/VuFindSearch/Backend/Solr/Connector.php

```diff
@@ -46,8 +47,6 @@
 use VuFindSearch\Backend\Solr\Document\AbstractDocument;
 use VuFindSearch\ParamBag;
 
-use VuFindSearch\Query\Query;
```

Comment:
I'm really puzzled why this `use` statement wasn't auto-removed by php-cs-fixer long ago. Am I correct in thinking that this was never needed, and you just cleaned it up as part of the other work here?

Reply:
Yes, at least it wasn't needed when I started working on the file. VSCode intelephense flagged it while php-cs-fixer didn't.

Reply:
Thanks for confirming!